l10n_eu_oss_oca/wizard/l10n_eu_oss_wizard.xml

```diff
@@ -10,7 +10,7 @@
         <field name="arch" type="xml">
             <form string='Setup EU OSS OCA Taxes'>
                 <group>
-                    <field name="company_id" groups="base.group_multi_company" />
+                    <field name="company_id" />
```

Comment:
Instead of removing the group, you have to keep it and add the same field as invisible without group.

Reply:
Thank you for the input :) It should be fixed now.